venv/pyvenv.cfg

```diff
@@ -0,0 +1,3 @@
+home = /Library/Frameworks/Python.framework/Versions/3.9/bin
+include-system-site-packages = false
+version = 3.9.5
```

Comment:
please remove this file
and/or add venv to https://github.com/plamere/spotipy/blob/master/.gitignore